### PR TITLE
WIP wicked: Simplify before test

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: tests/wicked/before_test.pm [hunk 1]
```diff
@@ -52,28 +52,22 @@ sub run {
...
-            zypper_call('--quiet in openvswitch', timeout => 200);
+            zypper_call('-q in openvswitch libteam-tools libteamdctl0 python-libteam', timeout => 200);
             assert_script_run("systemctl start openvswitch");
         }
     }
-    elsif (check_var('WICKED', '2nics')) {
-        assert_script_run('rm /etc/sysconfig/network/ifcfg-' . $ctx->iface());
-        assert_script_run("rcwickedd restart");
-    }
+    assert_script_run('rm /etc/sysconfig/network/ifcfg-' . $ctx->iface());
```
Comment (cfconrad) [inline tests/wicked/before_test.pm]: Does this  mean, that we now avoid having network at the beginning of each test?
And pre-run and post-run use `setup_static_network()` to do upload_logs and so on.

Comment (asmorodskyi) [reply tests/wicked/before_test.pm]: yes , I think  it is right approach . we already have a few scenarios where fact that eth0 has IP before test starts fail test case ... So I really waiting for offline upload/download feature to be done so we can do all thing in a right way :)

Comment (cfconrad) [reply tests/wicked/before_test.pm]: Ok I like the idea of getting a untouched network in each test.
What do you think about just doing `rm /etc/sysconfig/network/*` ?

Comment (cfconrad) [reply tests/wicked/before_test.pm]: We need to keep `/etc/sysconfig/network/config` but everything else...

File: lib/wickedbase.pm [hunk 2]
```diff
@@ -103,8 +103,8 @@ sub get_ip {
     my $ips_hash =
       {
         #                       SUT                       REF
-        host         => ['10.0.2.11/15',         '10.0.2.10/15'],
-        host6        => ['fd00:dead::2::11/64',  'fd00:dead::2::10/64'],
+        host         => ['10.0.2.11/24',         '10.0.2.10/24'],
+        host6        => ['2001:0db8:1234::000f', '2001:0db8:1234::000e'],
```
Comment (cfconrad) [inline lib/wickedbase.pm]: maybe we should not remove the netmask?

**Comment by asmorodskyi** [comment]
this one can be closed too :)

**Pull request closed without merging**